src/cpu.rs

```diff
@@ -75,6 +92,16 @@ impl Speed for CPU {
             Err(_) => eprint!("Failed"),
         }
     }
+    fn get_gov(&mut self) {
+        match self.read_value(
```

Comment:
This will always fail because read_value is only for i32 values. When it tries to read the file that contains the gov, it will try to read it into an i32 and break. You could either make a generic function for them (see display::print_vec) or just make another read_value function for string for temporary use.

Reply:
Once this is fixed i'll merge it